docs/intentReaderHandlers/RunTestHandler.md

```diff
@@ -0,0 +1,304 @@
+# RunTestHandler 
+
+## Overview
+
+RunTestHandler is invoked when the task specified in the intent has the value "runTest". This handler is used to run sanity tests in FCA. Whent this handler is invoked, it will loop through all the apis in openRpc schema, invoke the apis and perform schema validations on the responses. Once the execution is over, a test report is generated. For more about execution [here](../Execution.md#sanity-test-execution) and for reporting [here](../Reporting.md#supported-ways-of-retrieving-reports) 
```

Comment:
Typo Whent -> When

Also can we capitalize apis so it would be APIs

Reply:
Updated API in all handlers